setup.cfg

```diff
@@ -45,14 +46,15 @@ dev =
     mypy
     pip-tools
     pygithub
+    ghapi
     pymdown-extensions
-    pytest
+    pytest>=4
     pytest-timeout
     pyyaml
     requests
     typing-extensions
-    packaging>=20.8
     rich>=9.6
+    mypy>=0.800
```

Comment:
Is that mypy in the list twice now? Maybe we need to remove it on line 46?

Reply:
Oops! 😳